src/References/Mapping/Event/Adapter/ODM.php

```diff
@@ -32,7 +32,7 @@ public function getIdentifier($om, $object, $single = true)
         }
 
         if ($om instanceof EntityManagerInterface) {
-            if ($object instanceof ORMProxy) {
+            if ($object instanceof PersistenceProxy) {
```

Comment:
I'm wondering if relaxing this check deserves an entry in the changelog, since previous to this change only instances of `Doctrine\ORM\Proxy\Proxy` were admitted in this condition.

Reply:
I'd say this is more of an internal change to match the upstream API changes and not a deliberate feature change in this package.  So not necessarily something that needs to be called out in a changelog entry.